Dockerfile

```diff
@@ -64,4 +64,5 @@ ENV NODE_ENV=production
 ENV NODE_PATH=.
 
 USER node
-CMD [ "dumb-init", "node", "/home/node/app/src/server.js" ]
+
+CMD ["dumb-init", "node", "--experimental-loader", "newrelic/esm-loader.mjs", "-r", "newrelic", "/home/node/app/src/server.js"]
```

Comment:
can you also add this as a package.json command for local testing?

Reply:
yeah, new script

Reply:
added